### ISSUE TITLE
Suggestion: Discord Server For ngdevkit.

### ISSUE THREAD

**Issue by NeoGeoFreak2004** [dciabrin/ngdevkit#80]
hey, so anyways can you create a discord server or say a forum chat for ngdevkit? why do i say this, because there are no tutorials and help on the internet for the sdk.

**Comment by fab48** [comment]
Hi ! Ils easy to do ? Did you ? 
NGDK have one, maybe merge them ? (his sdk is incomplete and stop)
I can create it if you want.

**Comment by NeoGeoFreak2004** [comment]
Yes, sorry if I'm replying to you late, but yes, I want one created right now fab48. Thanks again and have a great day.

**Comment by dciabrin** [comment]
I'm closing this old one, as I am not a fan of discord and I don't have the bandwidth for a real-time chat channel.
I have opened github discussions though, so async discussion is now available.
